Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'devise'
```

Comment:
Add version to the gem (you can do that by coping the line directly from [ruby gems)](https://rubygems.org/gems/devise) that way you can always install compatible gem versions and nothing won’t break because of incompatibility. Also, since you are adding this gem only on `development` and `testing`, this gem would not be installed for example in production (this is the environment where once the app is tested, it's released to the public to be used) and your model user won't work.